soh/soh/SohMenuBar.cpp

```diff
@@ -1981,48 +1986,84 @@ void DrawRandomizerMenu() {
         ImGui::PushStyleVar(ImGuiStyleVar_ButtonTextAlign, ImVec2(0, 0));
         ImGui::PushStyleVar(ImGuiStyleVar_FrameBorderSize, 1.0f);
         ImGui::PushStyleColor(ImGuiCol_Border, ImVec4(0.22f, 0.38f, 0.56f, 1.0f));
+
     #ifdef __WIIU__
```

Comment:
Do we still want Wii U specific code on master now it has spun off to a branch/fork?

Reply:
No, this kind of Wii U specific stuff shouldn't be in main anymore.

Also, I don't like the way this button is done, I feel like it would wreak havoc with any auto scaling, but I'd have to test it first.

Reply:
But that's probably something I'd have to deal with later with the modern menu being applied here.

Reply:
Considering that the wiiu specific code is just for changing the size of stuff I don't see the harm on keeping it, I thought that the idea was to simply not worry about wiiu/switch stuff, not removing everything (specially if it doesn't increase complexity).

```cpp
#ifdef __WIIU__
    static ImVec2 buttonSize(200.0f * 2.0f, 0.0f);
    static ImVec2 buttonWithOptionsSize(170.0f * 2.0f, 0.0f);
    static ImVec2 optionsButtonSize(25.0f * 2.0f, 0.0f);
    static float separationToOptionsButton = 5.0f * 2.0f;
#else
    static ImVec2 buttonSize(200.0f, 0.0f);
    static ImVec2 buttonWithOptionsSize(170.0f, 0.0f);
    static ImVec2 optionsButtonSize(25.0f, 0.0f);
    static float separationToOptionsButton = 5.0f;
#endif
```

Reply:
You may be right about that. I know some Wii U specific code was removed at the time it was branched off, but I know not all of it was.